index.html

```diff
@@ -46,6 +46,157 @@ <h2>I'm a software Developer</h2>
+            <p class="description">
+              A daily selection of privately personalized reads;
+              no accounts or sign-ups required. 
+              has been the industry's standard
+            </p>
+            <ul class="tags-2">
+              <li class="lang-tag">html</li>
+              <li class="lang-tag">bootstrap</li>
+              <li class="lang-tag">Ruby</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+          </article>
+          <article class="project style2">
+            <h3 class="sub-heading2">Profesional Art Printing Data</h3>
+            <p class="description">
+              A daily selection of privately personalized reads;
+              no accounts or sign-ups required. 
+              has been the industry's standard
+            </p>
+            <ul class="tags-2">
+              <li class="lang-tag">html</li>
+              <li class="lang-tag">bootstrap</li>
+              <li class="lang-tag">Ruby</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+          </article>
+          <article class="project style2">
+            <h3 class="sub-heading2">Profesional Art Printing Data</h3>
+            <p class="description">
+              A daily selection of privately personalized reads;
+              no accounts or sign-ups required. 
+              has been the industry's standard
+            </p>
+            <ul class="tags-2">
+              <li class="lang-tag">html</li>
+              <li class="lang-tag">bootstrap</li>
+              <li class="lang-tag">Ruby</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+          </article>
+          <article class="project style2">
+            <h3 class="sub-heading2">Profesional Art Printing Data</h3>
+            <p class="description">
+              A daily selection of privately personalized reads;
+              no accounts or sign-ups required. 
+              has been the industry's standard
+            </p>
+            <ul class="tags-2">
+              <li class="lang-tag">html</li>
+              <li class="lang-tag">bootstrap</li>
+              <li class="lang-tag">Ruby</li>
+            </ul>
+            <button class="btn" type="button">See Project</button>
+          </article>
+        </div>
+      </section>
+      <section class="aboutme">
+        <div class="text-container">
+          <h2>About me</h2>
+          <p>
+            Hello I'm a software developer! I can help you build a product,
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don't hestiate to contact me.
+          </p>
+          <button type="button" class="btn">Get My Resume</button>
+        </div>
+        <hr class="line">
+        <div class="languages-container">
```

Comment:
As stated in the [requirements for this milestone](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), please wrap the technologies' categories in a `<ul>` tag, so in this case, the children or `<li>` will be the `languages-container` :ballot_box_with_check: